rollbar-reactive-streams-reactor/src/test/java/com/rollbar/reactivestreams/notifier/ReactorRollbarTest.java

```diff
@@ -0,0 +1,136 @@
+package com.rollbar.reactivestreams.notifier;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.rollbar.api.payload.Payload;
+import com.rollbar.api.payload.data.body.BodyContent;
+import com.rollbar.api.payload.data.body.Frame;
+import com.rollbar.api.payload.data.body.Trace;
+import com.rollbar.notifier.sender.result.Response;
+import com.rollbar.reactivestreams.notifier.config.Config;
+import com.rollbar.reactivestreams.notifier.config.ConfigBuilder;
+import com.rollbar.reactivestreams.notifier.sender.AsyncSender;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.util.List;
+
+public class ReactorRollbarTest {
+  private ReactorRollbar rollbar;
+  private AsyncSender sender;
+
+  @BeforeEach
+  private void setUp() {
+    sender = mock(AsyncSender.class);
+    Response response = mock(Response.class);
+    when(sender.send(any())).thenReturn(Mono.just(response));
+
+    Config config = ConfigBuilder
+        .withAccessToken("TEST TOKEN")
+        .sender(sender)
+        .environment("development")
+        .build();
+
+    rollbar = new ReactorRollbar(config);
+  }
+
+  @Test
+  public void onMonoErrorShouldReportMonoErrors() {
+    Mono<String> sut = Mono.<String>error(new TestException("Test Exception"))
+        .onErrorResume(rollbar::logMonoError);
+
+    StepVerifier.create(sut)
+        .expectError(TestException.class)
+        .verify();
+
+    verify(sender, times(1)).send(any());
+  }
+
+  @Test
+  public void onMonoSuccessShouldReturnMonoValue() {
+    Mono<String> sut = Mono.just("successValue")
+        .onErrorResume(rollbar::logMonoError);
+
+    StepVerifier.create(sut)
+        .expectNext("successValue")
+        .expectComplete()
+        .verify();
+
+    verify(sender, times(0)).send(any());
+  }
```

Comment:
The test name doesn't correspond to the test body itself. I would rename it to `onMonoSuccessShouldNotReportMonoErrors()` and maybe keep the `onMonoSuccessShouldReturnMonoValue()` with updated test to actually verify the returned value.